src/main/kotlin/no/fdk/fdk_reasoning_service/config/CircuitBreakerConsumerConfiguration.kt

```diff
@@ -0,0 +1,44 @@
+package no.fdk.fdk_reasoning_service.config
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker.StateTransition
+import io.github.resilience4j.circuitbreaker.CircuitBreakerRegistry
+import io.github.resilience4j.circuitbreaker.event.CircuitBreakerOnStateTransitionEvent
+import no.fdk.fdk_reasoning_service.kafka.KafkaHarvestedEventConsumer
+import no.fdk.fdk_reasoning_service.kafka.KafkaManager
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+open class CircuitBreakerConsumerConfiguration(
+    private val circuitBreakerRegistry: CircuitBreakerRegistry,
+    private val kafkaManager: KafkaManager
+) {
+    init {
+        LOGGER.debug("Configuring circuit breaker event listener")
+        circuitBreakerRegistry.circuitBreaker(KafkaHarvestedEventConsumer.REASONING_LISTENER_ID)
```

Comment:
Selv om verdien er det samme så må du skille mellom KafkaListener og CircuitBreaker ID.

Reply:
For dette er CircuitBreakerId, ja. Legg`CIRCUIT_BREAKER_ID` inn som const i companion object til denne klassen, så er det lik struktur 👍

Reply:
Ah, bra catch, ikke meningen at den skulle dit.